index.html

```diff
@@ -212,18 +219,21 @@ <h3>Skills</h3>
         </ul>
     </section>
     <footer>
-        <h4>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h4>
-        <form action="https://formspree.io/f/xlezzzro" method="POST">
+        <div class="form-title">
+            <h4>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h4>
+            <form action="https://formspree.io/f/xlezzzro" method="POST">
+
+                <input type="text" name="full-name" required="required" maxlength="30" placeholder="Full Name">
+
+                <input type="email" name="email" required="required" placeholder="Email address">
 
-            <input type="text" name="full-name" required="required" maxlength="30" placeholder="Full Name">
-    
-            <input type="email" name="email" required="required" placeholder="Email address">
-    
-            <textarea name="message" maxlength="500" placeholder="Enter text here" required></textarea>
-    
-            <button type="submit">Get in touch</button>
-        </form>
```

Comment:
- [ ] kindly add a little bit of padding to your desktop form placeholders so the get the position of the original figma version